willisapi_client/services/metadata/archive.py

```
@@ -0,0 +1,83 @@
+
+    All failures are logged and swallowed — archiving must never abort the
+    actual data upload.
+    """
+    try:
+        wc = WillisapiClient(env=env)
+        payload = {
+            "filename": os.path.basename(csv_path),
+            "total_rows": total_rows,
+            "upload_type": upload_type,
+        }
+        res = requests.post(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)